EAGLE Libraries/HyTechDevices.lbr

```diff
@@ -16604,6 +16769,82 @@ Demo Board</text>
 <text x="0" y="23.368" size="1.778" layer="95">&gt;NAME</text>
 <text x="0" y="-2.032" size="1.778" layer="95">&gt;VALUE</text>
 </symbol>
+<symbol name="RES3">
```

Comment:
Could you use a more descriptive name for the symbol cause "RES3" doesn't explain it very well.
In addition, you seem to have lost the symbol Leo made for his resistor array.